source/game.person.bmx

```diff
@@ -643,19 +665,28 @@ Type TPersonProductionData Extends TPersonProductionBaseData
 		If Not PersonsGainExperienceForProgrammes Then Return
 
 		'gain experience for each done job
+		'TODO update not intuitive on game start for insignificant persons
+		'even if they are castable and have a role in an existing licence
+		'they gain experience only after the third production
+		'(Jonas Becker no job experience after producing a series...)
```

Comment:
We could bring in some XP for the older productions once they become "attributable" (in the sense of worth to mention their name).
The original cause for not giving XP etc earlier was to avoid the whole "production"-extra-data for persons only being used 1-2 times in a cast.

Reply:
... Es sieht aber so aus, als würden diese Daten schon jetzt irgendwie erhoben. Zumindest die Liste der früheren Produktionen wird eben schon gefüllt. Man müsste vielleicht mal schauen, wieviele Insignificant+Castables unterhalb der Schwelle liegen. Es wäre verständlich, wenn 1000 von 1050 Personen "unnütze" Daten haben. Wenn es aber nur 50 von 1050 betrifft, wäre Konsistenz hingegen schöner, da die Speicherersparnis vernachlässigbar ist.

Reply:
Ich denke, diesen Punkt muss ich mir für später aufheben. Eine erste oberflächliche Analyse zeigt kein eindeutiges Bild. Scheinbar gibt es überhaupt vergleichsweise relativ wenige fiktive Personen mit Produktionen, von denen viele aber auch erst lange nach Spielstart veröffentlicht werden (relative year). Manche sammeln also auch noch während des Spiels "automatisch" an Erfahrung.

Interessant war auch, dass (im Vergleich) sehr viele nicht-fiktionale Personen Attribute persistiert bekommen, obwohl sie ja nie gecastet werden können.

Reply:
Man koennte ja einen debugstop oder log einbauen, wenn ein nichtfiktionaler Attribute bekommt.

Evtl ist ja eine der Programmdaten falsch geflaggt ..als customproduction oder so.